2023-10-16 00:47:49,131 - WARNING -  * Debugger is active!
2023-10-16 00:47:49,132 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 00:48:58,304 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,305 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,312 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,423 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,428 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,432 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,670 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,670 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:58,678 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/session.py', reloading
2023-10-16 00:48:59,280 - WARNING -  * Debugger is active!
2023-10-16 00:48:59,281 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 00:49:19,231 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 00:49:19,232 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 00:49:19,238 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 00:49:19,577 - WARNING -  * Debugger is active!
2023-10-16 00:49:19,577 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 00:50:08,430 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 00:50:08,430 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 00:50:08,434 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 00:50:08,787 - WARNING -  * Debugger is active!
2023-10-16 00:50:08,787 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 00:50:10,260 - INFO - 127.0.0.1 - - [16/Oct/2023 00:50:10] "[33mGET /log_monitor HTTP/1.1[0m" 404 -
2023-10-16 00:50:11,637 - INFO - 127.0.0.1 - - [16/Oct/2023 00:50:11] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2023-10-16 00:50:11,638 - INFO - 127.0.0.1 - - [16/Oct/2023 00:50:11] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2023-10-16 00:50:11,839 - INFO - 127.0.0.1 - - [16/Oct/2023 00:50:11] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 00:50:11,954 - INFO - 127.0.0.1 - - [16/Oct/2023 00:50:11] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 00:57:06,402 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 00:57:06,402 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 00:57:06,451 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 00:57:06,876 - WARNING -  * Debugger is active!
2023-10-16 00:57:06,880 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 01:03:23,437 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 01:03:23,438 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 01:03:23,458 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 01:03:24,591 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 01:03:24,891 - WARNING -  * Debugger is active!
2023-10-16 01:03:24,892 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 01:03:28,623 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 01:03:28,624 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 01:03:28,628 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 01:03:28,953 - WARNING -  * Debugger is active!
2023-10-16 01:03:28,953 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 01:03:40,291 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 01:03:40,291 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 01:03:40,299 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 01:03:40,615 - WARNING -  * Debugger is active!
2023-10-16 01:03:40,615 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 01:04:21,807 - INFO - 127.0.0.1 - - [16/Oct/2023 01:04:21] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 01:04:42,543 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 01:04:42,543 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 01:04:42,548 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 01:04:42,856 - WARNING -  * Debugger is active!
2023-10-16 01:04:42,856 - INFO -  * Debugger PIN: 262-856-221
2023-10-16 10:42:31,270 - INFO - 127.0.0.1 - - [16/Oct/2023 10:42:31] "[36mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2023-10-16 10:42:31,271 - INFO - 127.0.0.1 - - [16/Oct/2023 10:42:31] "[36mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2023-10-16 10:42:31,730 - INFO - 127.0.0.1 - - [16/Oct/2023 10:42:31] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 10:42:31,819 - INFO - 127.0.0.1 - - [16/Oct/2023 10:42:31] "[36mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 14:22:17,763 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 14:22:17,764 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 14:22:17,787 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 14:22:18,406 - WARNING -  * Debugger is active!
2023-10-16 14:22:18,406 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 14:22:21,777 - INFO - DB session created successfully.
2023-10-16 14:22:21,778 - INFO - DB session closed successfully.
2023-10-16 14:22:21,805 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 14:22:21,806 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 14:22:21,806 - INFO - 127.0.0.1 - - [16/Oct/2023 14:22:21] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 14:22:22,073 - INFO - 127.0.0.1 - - [16/Oct/2023 14:22:22] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2023-10-16 14:22:38,417 - INFO - DB session created successfully.
2023-10-16 14:22:38,417 - INFO - DB session closed successfully.
2023-10-16 14:22:38,418 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 14:22:38,418 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 14:22:38,419 - INFO - 127.0.0.1 - - [16/Oct/2023 14:22:38] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 14:22:43,417 - INFO - 127.0.0.1 - - [16/Oct/2023 14:22:43] "[33mGET /monitor HTTP/1.1[0m" 404 -
2023-10-16 14:22:49,151 - INFO - 127.0.0.1 - - [16/Oct/2023 14:22:49] "GET /log_files HTTP/1.1" 200 -
2023-10-16 14:29:28,738 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 14:29:28,748 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 14:29:28,761 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 14:29:28,910 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 14:29:29,104 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 14:29:29,490 - INFO - Starting the server.
2023-10-16 14:29:29,494 - WARNING -  * Debugger is active!
2023-10-16 14:29:29,495 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:02:33,093 - INFO - Endpoint '/' accessed.
2023-10-16 15:02:33,102 - INFO - DB session created successfully.
2023-10-16 15:02:33,103 - INFO - DB session closed successfully.
2023-10-16 15:02:33,137 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:02:33,137 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:02:33,137 - INFO - 127.0.0.1 - - [16/Oct/2023 15:02:33] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:02:45,290 - INFO - Endpoint '/log_files' accessed.
2023-10-16 15:02:45,311 - INFO - 127.0.0.1 - - [16/Oct/2023 15:02:45] "GET /log_files HTTP/1.1" 200 -
2023-10-16 15:07:26,834 - INFO - 127.0.0.1 - - [16/Oct/2023 15:07:26] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2023-10-16 15:08:53,116 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:08:53,168 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:08:53,217 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:08:54,216 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:08:55,343 - INFO - Starting the server.
2023-10-16 15:08:55,352 - WARNING -  * Debugger is active!
2023-10-16 15:08:55,353 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:08:59,273 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:08:59,273 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:08:59,288 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:08:59,737 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:09:00,287 - INFO - Starting the server.
2023-10-16 15:09:00,290 - WARNING -  * Debugger is active!
2023-10-16 15:09:00,290 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:09:34,206 - INFO - Server stopped.
2023-10-16 15:09:38,543 - INFO - Starting the server.
2023-10-16 15:09:38,567 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:09:38,571 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:09:38,616 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:09:39,644 - INFO - Starting the server.
2023-10-16 15:09:39,677 - WARNING -  * Debugger is active!
2023-10-16 15:09:39,678 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:09:46,742 - INFO - Endpoint '/' accessed.
2023-10-16 15:09:46,743 - INFO - DB session created successfully.
2023-10-16 15:09:46,743 - INFO - DB session closed successfully.
2023-10-16 15:09:46,753 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:09:46,760 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:09:46,761 - INFO - 127.0.0.1 - - [16/Oct/2023 15:09:46] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:24:13,319 - INFO - Server stopped.
2023-10-16 15:24:13,602 - INFO - Server stopped.
2023-10-16 15:30:21,084 - INFO - Testing logger configuration in queries.py
2023-10-16 15:30:21,088 - INFO - Starting the server.
2023-10-16 15:30:21,107 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:30:21,107 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:30:21,128 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:30:21,582 - INFO - Testing logger configuration in queries.py
2023-10-16 15:30:21,586 - INFO - Starting the server.
2023-10-16 15:30:21,590 - WARNING -  * Debugger is active!
2023-10-16 15:30:21,594 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:30:28,288 - INFO - Endpoint '/' accessed.
2023-10-16 15:30:28,289 - INFO - DB session created successfully.
2023-10-16 15:30:28,289 - INFO - DB session closed successfully.
2023-10-16 15:30:28,413 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:30:28,414 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:30:28,415 - INFO - 127.0.0.1 - - [16/Oct/2023 15:30:28] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:34:57,668 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 15:34:57,668 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 15:34:57,672 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 15:34:58,013 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:34:58,292 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:34:58,295 - INFO - Starting the server.
2023-10-16 15:34:58,297 - WARNING -  * Debugger is active!
2023-10-16 15:34:58,298 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:35:02,407 - INFO - Server stopped.
2023-10-16 15:35:02,595 - INFO - Server stopped.
2023-10-16 15:35:05,489 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:35:05,491 - INFO - Starting the server.
2023-10-16 15:35:05,495 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:35:05,495 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:35:05,500 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:35:05,939 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:35:05,943 - INFO - Starting the server.
2023-10-16 15:35:05,947 - WARNING -  * Debugger is active!
2023-10-16 15:35:05,948 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:35:11,565 - INFO - Endpoint '/' accessed.
2023-10-16 15:35:11,565 - INFO - Function 'get_all_messages()' called.
2023-10-16 15:35:11,565 - INFO - DB session created successfully.
2023-10-16 15:35:11,565 - INFO - DB session closed successfully.
2023-10-16 15:35:11,565 - INFO - Successfully connected to the database.
2023-10-16 15:35:11,577 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:35:11,578 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:35:11,579 - INFO - 127.0.0.1 - - [16/Oct/2023 15:35:11] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:37:04,441 - INFO - Server stopped.
2023-10-16 15:37:04,554 - INFO - Server stopped.
2023-10-16 15:37:07,102 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:37:07,104 - INFO - Starting the server.
2023-10-16 15:37:07,108 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:37:07,108 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:37:07,113 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:37:07,709 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:37:07,713 - INFO - Starting the server.
2023-10-16 15:37:07,717 - WARNING -  * Debugger is active!
2023-10-16 15:37:07,717 - INFO -  * Debugger PIN: 748-054-032
2023-10-16 15:37:15,744 - INFO - Endpoint '/' accessed.
2023-10-16 15:37:15,746 - INFO - Function 'get_all_messages()' called.
2023-10-16 15:37:15,746 - INFO - DB session created successfully.
2023-10-16 15:37:15,747 - INFO - DB session closed successfully.
2023-10-16 15:37:15,747 - INFO - Successfully connected to the database.
2023-10-16 15:37:15,759 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:37:15,760 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:37:15,760 - INFO - 127.0.0.1 - - [16/Oct/2023 15:37:15] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:37:37,745 - INFO - Endpoint '/log_files' accessed.
2023-10-16 15:37:37,749 - INFO - 127.0.0.1 - - [16/Oct/2023 15:37:37] "GET /log_files HTTP/1.1" 200 -
2023-10-16 15:38:25,710 - INFO - Server stopped.
2023-10-16 15:38:25,873 - INFO - Server stopped.
2023-10-16 15:44:45,177 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:44:45,183 - INFO - Starting the server.
2023-10-16 15:44:45,197 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:44:45,197 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:44:45,221 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:44:45,973 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:44:45,976 - INFO - Starting the server.
2023-10-16 15:44:45,979 - WARNING -  * Debugger is active!
2023-10-16 15:44:45,984 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 15:44:50,176 - INFO - Endpoint '/' accessed.
2023-10-16 15:44:50,177 - INFO - Function 'get_all_messages()' called.
2023-10-16 15:44:50,177 - INFO - DB session created successfully.
2023-10-16 15:44:50,177 - INFO - DB session closed successfully.
2023-10-16 15:44:50,178 - INFO - Successfully connected to the database.
2023-10-16 15:44:50,232 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:44:50,232 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:44:50,233 - INFO - 127.0.0.1 - - [16/Oct/2023 15:44:50] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:44:50,972 - INFO - 127.0.0.1 - - [16/Oct/2023 15:44:50] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2023-10-16 15:53:46,763 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:53:46,771 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:53:46,772 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:53:47,586 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:53:48,769 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:53:58,148 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:53:58,151 - INFO - Starting the server.
2023-10-16 15:53:58,162 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:53:58,162 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:53:58,200 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:53:58,497 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:53:58,500 - INFO - Starting the server.
2023-10-16 15:53:58,502 - WARNING -  * Debugger is active!
2023-10-16 15:53:58,502 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 15:54:11,773 - INFO - Endpoint '/' accessed.
2023-10-16 15:54:11,774 - INFO - Function 'get_all_messages()' called.
2023-10-16 15:54:11,774 - INFO - DB session created successfully.
2023-10-16 15:54:11,774 - INFO - DB session closed successfully.
2023-10-16 15:54:11,774 - INFO - Successfully connected to the database.
2023-10-16 15:54:11,795 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:54:11,795 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:54:11,796 - INFO - 127.0.0.1 - - [16/Oct/2023 15:54:11] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:54:54,962 - INFO - Endpoint '/' accessed.
2023-10-16 15:54:54,963 - INFO - Function 'get_all_messages()' called.
2023-10-16 15:54:54,963 - INFO - DB session created successfully.
2023-10-16 15:54:54,963 - INFO - DB session closed successfully.
2023-10-16 15:54:54,963 - INFO - Successfully connected to the database.
2023-10-16 15:54:54,964 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 15:54:54,964 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 15:54:54,964 - INFO - 127.0.0.1 - - [16/Oct/2023 15:54:54] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 15:55:56,137 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:55:56,150 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:55:56,181 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 15:55:56,646 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:55:56,988 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:55:56,993 - INFO - Starting the server.
2023-10-16 15:55:56,997 - WARNING -  * Debugger is active!
2023-10-16 15:55:56,997 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 15:55:58,665 - INFO - Server stopped.
2023-10-16 15:55:58,785 - INFO - Server stopped.
2023-10-16 15:55:59,766 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:55:59,769 - INFO - Starting the server.
2023-10-16 15:55:59,773 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 15:55:59,773 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 15:55:59,777 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 15:56:00,085 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 15:56:00,089 - INFO - Starting the server.
2023-10-16 15:56:00,093 - WARNING -  * Debugger is active!
2023-10-16 15:56:00,093 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:00:53,648 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:00:53,648 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:00:53,656 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:00:54,114 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:00:54,939 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:00:54,944 - INFO - Starting the server.
2023-10-16 16:00:54,948 - WARNING -  * Debugger is active!
2023-10-16 16:00:54,948 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:01:05,943 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 16:01:05,944 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 16:01:05,944 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 16:01:06,092 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 16:01:06,108 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 16:01:06,114 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-16 16:01:06,487 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:01:06,867 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:01:06,870 - INFO - Starting the server.
2023-10-16 16:01:06,873 - WARNING -  * Debugger is active!
2023-10-16 16:01:06,873 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:01:09,529 - INFO - Server stopped.
2023-10-16 16:01:09,658 - INFO - Server stopped.
2023-10-16 16:01:11,089 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:01:11,092 - INFO - Starting the server.
2023-10-16 16:01:11,095 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 16:01:11,095 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 16:01:11,100 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:01:11,388 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:01:11,391 - INFO - Starting the server.
2023-10-16 16:01:11,393 - WARNING -  * Debugger is active!
2023-10-16 16:01:11,394 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:01:18,703 - INFO - Endpoint '/' accessed.
2023-10-16 16:01:18,704 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:01:18,704 - INFO - DB session created successfully.
2023-10-16 16:01:18,704 - INFO - DB session closed successfully.
2023-10-16 16:01:18,704 - INFO - Successfully connected to the database.
2023-10-16 16:01:18,716 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:01:18,716 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:01:18,716 - INFO - 127.0.0.1 - - [16/Oct/2023 16:01:18] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 16:01:34,631 - INFO - Endpoint '/log_files' accessed.
2023-10-16 16:01:34,642 - INFO - 127.0.0.1 - - [16/Oct/2023 16:01:34] "GET /log_files HTTP/1.1" 200 -
2023-10-16 16:01:38,594 - INFO - Endpoint '/' accessed.
2023-10-16 16:01:38,594 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:01:38,595 - INFO - DB session created successfully.
2023-10-16 16:01:38,595 - INFO - DB session closed successfully.
2023-10-16 16:01:38,595 - INFO - Successfully connected to the database.
2023-10-16 16:01:38,596 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:01:38,596 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:01:38,597 - INFO - 127.0.0.1 - - [16/Oct/2023 16:01:38] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 16:03:15,693 - INFO - Server stopped.
2023-10-16 16:03:15,924 - INFO - Server stopped.
2023-10-16 16:04:13,694 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:04:13,698 - INFO - Starting the server.
2023-10-16 16:04:13,704 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 16:04:13,705 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 16:04:13,712 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:04:14,174 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:04:14,178 - INFO - Starting the server.
2023-10-16 16:04:14,182 - WARNING -  * Debugger is active!
2023-10-16 16:04:14,182 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:04:21,814 - INFO - Endpoint '/' accessed.
2023-10-16 16:04:21,814 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:04:21,815 - INFO - DB session created successfully.
2023-10-16 16:04:21,816 - INFO - DB session closed successfully.
2023-10-16 16:04:21,817 - INFO - Successfully connected to the database.
2023-10-16 16:04:21,892 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:04:21,892 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:04:21,893 - INFO - 127.0.0.1 - - [16/Oct/2023 16:04:21] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 16:04:26,423 - INFO - Endpoint '/log_files' accessed.
2023-10-16 16:04:26,453 - INFO - 127.0.0.1 - - [16/Oct/2023 16:04:26] "[35m[1mGET /log_files HTTP/1.1[0m" 500 -
2023-10-16 16:04:26,530 - INFO - 127.0.0.1 - - [16/Oct/2023 16:04:26] "[36mGET /log_files?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2023-10-16 16:04:26,534 - INFO - 127.0.0.1 - - [16/Oct/2023 16:04:26] "[36mGET /log_files?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2023-10-16 16:04:26,746 - INFO - 127.0.0.1 - - [16/Oct/2023 16:04:26] "[36mGET /log_files?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 16:04:26,796 - INFO - 127.0.0.1 - - [16/Oct/2023 16:04:26] "[36mGET /log_files?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2023-10-16 16:04:58,523 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:04:58,524 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:04:58,529 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:04:58,680 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:04:58,684 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:04:58,701 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:04:58,884 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:04:59,354 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:04:59,359 - INFO - Starting the server.
2023-10-16 16:04:59,363 - WARNING -  * Debugger is active!
2023-10-16 16:04:59,364 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:05:03,689 - INFO - Endpoint '/log_files' accessed.
2023-10-16 16:05:03,705 - INFO - 127.0.0.1 - - [16/Oct/2023 16:05:03] "GET /log_files HTTP/1.1" 200 -
2023-10-16 16:10:32,809 - INFO - Server stopped.
2023-10-16 16:10:32,918 - INFO - Server stopped.
2023-10-16 16:10:34,657 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:10:34,660 - INFO - Starting the server.
2023-10-16 16:10:34,664 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 16:10:34,664 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 16:10:34,668 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:10:34,971 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:10:34,973 - INFO - Starting the server.
2023-10-16 16:10:34,976 - WARNING -  * Debugger is active!
2023-10-16 16:10:34,976 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:10:40,545 - INFO - Endpoint '/' accessed.
2023-10-16 16:10:40,545 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:10:40,545 - INFO - DB session created successfully.
2023-10-16 16:10:40,546 - INFO - DB session closed successfully.
2023-10-16 16:10:40,546 - INFO - Successfully connected to the database.
2023-10-16 16:10:40,563 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:10:40,563 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:10:40,564 - INFO - 127.0.0.1 - - [16/Oct/2023 16:10:40] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 16:10:46,912 - INFO - Endpoint '/log_files' accessed.
2023-10-16 16:10:46,956 - INFO - 127.0.0.1 - - [16/Oct/2023 16:10:46] "GET /log_files HTTP/1.1" 200 -
2023-10-16 16:10:50,613 - INFO - Endpoint '/' accessed.
2023-10-16 16:10:50,613 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:10:50,614 - INFO - DB session created successfully.
2023-10-16 16:10:50,614 - INFO - DB session closed successfully.
2023-10-16 16:10:50,614 - INFO - Successfully connected to the database.
2023-10-16 16:10:50,618 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:10:50,619 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:10:50,619 - INFO - 127.0.0.1 - - [16/Oct/2023 16:10:50] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 16:32:37,016 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:32:37,019 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:32:37,040 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:32:37,201 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:32:37,208 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:32:37,213 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:32:37,554 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:32:38,306 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:32:38,308 - INFO - Starting the server.
2023-10-16 16:32:38,314 - WARNING -  * Debugger is active!
2023-10-16 16:32:38,315 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:32:41,249 - INFO - Server stopped.
2023-10-16 16:32:41,251 - INFO - Starting the server.
2023-10-16 16:32:41,251 - WARNING -  * Debugger is active!
2023-10-16 16:32:41,251 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:32:41,357 - INFO - Server stopped.
2023-10-16 16:33:08,654 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:33:08,658 - INFO - Starting the server.
2023-10-16 16:33:08,668 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 16:33:08,669 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 16:33:08,673 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:33:08,982 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:33:08,986 - INFO - Starting the server.
2023-10-16 16:33:08,988 - WARNING -  * Debugger is active!
2023-10-16 16:33:08,988 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:33:16,557 - INFO - Endpoint '/' accessed.
2023-10-16 16:33:16,560 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:33:16,561 - INFO - DB session created successfully.
2023-10-16 16:33:16,564 - INFO - DB session closed successfully.
2023-10-16 16:33:16,565 - INFO - Successfully connected to the database.
2023-10-16 16:33:16,629 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:33:16,631 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:33:16,634 - INFO - 127.0.0.1 - - [16/Oct/2023 16:33:16] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 16:33:29,765 - INFO - Server stopped.
2023-10-16 16:33:29,766 - INFO - Starting the server.
2023-10-16 16:33:29,767 - WARNING -  * Debugger is active!
2023-10-16 16:33:29,767 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:33:29,872 - INFO - Server stopped.
2023-10-16 16:33:29,874 - INFO - Starting the server.
2023-10-16 16:33:29,874 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 16:33:29,875 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 16:33:29,875 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:33:30,211 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:33:30,214 - INFO - Starting the server.
2023-10-16 16:33:30,216 - WARNING -  * Debugger is active!
2023-10-16 16:33:30,217 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:33:49,896 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:49,919 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:49,952 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,024 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,024 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,032 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,298 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,300 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,302 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,312 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,490 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,494 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,508 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/flask_app/app.py', reloading
2023-10-16 16:33:50,913 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:33:51,688 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:33:51,698 - INFO - Starting the server.
2023-10-16 16:33:51,701 - WARNING -  * Debugger is active!
2023-10-16 16:33:51,702 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:33:55,960 - INFO - Server stopped.
2023-10-16 16:33:55,962 - INFO - Starting the server.
2023-10-16 16:33:55,967 - WARNING -  * Debugger is active!
2023-10-16 16:33:55,968 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:33:56,123 - INFO - Server stopped.
2023-10-16 16:33:58,364 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:33:58,367 - INFO - Starting the server.
2023-10-16 16:33:58,373 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 16:33:58,374 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 16:33:58,382 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 16:33:58,907 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 16:33:58,911 - INFO - Starting the server.
2023-10-16 16:33:58,915 - WARNING -  * Debugger is active!
2023-10-16 16:33:58,915 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 16:34:07,249 - INFO - Endpoint '/' accessed.
2023-10-16 16:34:07,250 - INFO - Function 'get_all_messages()' called.
2023-10-16 16:34:07,250 - INFO - DB session created successfully.
2023-10-16 16:34:07,250 - INFO - DB session closed successfully.
2023-10-16 16:34:07,250 - INFO - Successfully connected to the database.
2023-10-16 16:34:07,261 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-16 16:34:07,263 - ERROR - Problem retrieving messages or columns from the database.
2023-10-16 16:34:07,263 - INFO - 127.0.0.1 - - [16/Oct/2023 16:34:07] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-16 17:37:54,004 - INFO - Server stopped.
2023-10-16 17:37:54,038 - INFO - Starting the server.
2023-10-16 17:37:54,059 - WARNING -  * Debugger is active!
2023-10-16 17:37:54,074 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 17:37:54,254 - INFO - Server stopped.
2023-10-16 17:37:54,569 - INFO - Starting the server.
2023-10-16 17:37:54,830 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-16 17:37:54,830 - INFO - [33mPress CTRL+C to quit[0m
2023-10-16 17:37:54,835 - INFO -  * Restarting with watchdog (inotify)
2023-10-16 17:37:55,848 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-16 17:37:55,853 - INFO - Starting the server.
2023-10-16 17:37:55,869 - WARNING -  * Debugger is active!
2023-10-16 17:37:55,871 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 17:37:57,321 - INFO - Server stopped.
2023-10-16 17:37:57,330 - INFO - Starting the server.
2023-10-16 17:37:57,330 - WARNING -  * Debugger is active!
2023-10-16 17:37:57,330 - INFO -  * Debugger PIN: 685-590-235
2023-10-16 17:37:57,574 - INFO - Server stopped.
2023-10-17 00:38:19,981 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 00:38:19,984 - INFO - Starting the server.
2023-10-17 00:38:19,995 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-17 00:38:19,995 - INFO - [33mPress CTRL+C to quit[0m
2023-10-17 00:38:20,017 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 00:38:20,500 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 00:38:20,505 - INFO - Starting the server.
2023-10-17 00:38:20,510 - WARNING -  * Debugger is active!
2023-10-17 00:38:20,511 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 00:38:33,208 - INFO - Server stopped.
2023-10-17 00:38:33,406 - INFO - Server stopped.
2023-10-17 00:43:02,623 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 00:43:02,626 - INFO - Starting the server.
2023-10-17 00:43:02,630 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-17 00:43:02,630 - INFO - [33mPress CTRL+C to quit[0m
2023-10-17 00:43:02,635 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 00:43:02,989 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 00:43:02,992 - INFO - Starting the server.
2023-10-17 00:43:02,995 - WARNING -  * Debugger is active!
2023-10-17 00:43:02,995 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 00:43:12,423 - INFO - Endpoint '/' accessed.
2023-10-17 00:43:12,443 - INFO - Function 'get_all_messages()' called.
2023-10-17 00:43:12,451 - INFO - DB session created successfully.
2023-10-17 00:43:12,451 - INFO - DB session closed successfully.
2023-10-17 00:43:12,451 - INFO - Successfully connected to the database.
2023-10-17 00:43:12,473 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-17 00:43:12,474 - ERROR - Problem retrieving messages or columns from the database.
2023-10-17 00:43:12,474 - INFO - 127.0.0.1 - - [17/Oct/2023 00:43:12] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-17 01:09:26,407 - INFO - Server stopped.
2023-10-17 01:09:26,614 - INFO - Server stopped.
2023-10-17 01:13:46,444 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 01:13:46,447 - INFO - Starting the server.
2023-10-17 01:13:46,451 - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-10-17 01:13:46,451 - INFO - [33mPress CTRL+C to quit[0m
2023-10-17 01:13:46,455 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 01:13:46,770 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 01:13:46,773 - INFO - Starting the server.
2023-10-17 01:13:46,775 - WARNING -  * Debugger is active!
2023-10-17 01:13:46,776 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 01:14:00,873 - INFO - Endpoint '/' accessed.
2023-10-17 01:14:00,873 - INFO - Function 'get_all_messages()' called.
2023-10-17 01:14:00,873 - INFO - DB session created successfully.
2023-10-17 01:14:00,873 - INFO - DB session closed successfully.
2023-10-17 01:14:00,873 - INFO - Successfully connected to the database.
2023-10-17 01:14:00,902 - ERROR - Error occurred while fetching messages: (sqlite3.OperationalError) no such table: messages
[SQL: SELECT messages.id AS messages_id, messages.source AS messages_source, messages.subject AS messages_subject, messages.sender AS messages_sender, messages.recipients AS messages_recipients, messages.content AS messages_content, messages.footer AS messages_footer, messages.original_content AS messages_original_content, messages.timestamp AS messages_timestamp, messages.attachments_path AS messages_attachments_path 
FROM messages]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-10-17 01:14:00,902 - ERROR - Problem retrieving messages or columns from the database.
2023-10-17 01:14:00,905 - INFO - 127.0.0.1 - - [17/Oct/2023 01:14:00] "[35m[1mGET / HTTP/1.1[0m" 500 -
2023-10-17 11:42:20,212 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-17 11:42:20,230 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-17 11:42:20,244 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-17 11:42:20,390 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-17 11:42:20,398 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-17 11:42:20,405 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/database/operations/queries.py', reloading
2023-10-17 11:42:20,739 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 11:42:21,831 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 11:42:21,834 - INFO - Starting the server.
2023-10-17 11:42:21,839 - WARNING -  * Debugger is active!
2023-10-17 11:42:21,840 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 11:57:32,830 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/config/config.py', reloading
2023-10-17 11:57:32,831 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/config/config.py', reloading
2023-10-17 11:57:32,838 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/config/config.py', reloading
2023-10-17 11:57:33,154 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 11:57:33,612 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 11:57:33,615 - INFO - Starting the server.
2023-10-17 11:57:33,619 - WARNING -  * Debugger is active!
2023-10-17 11:57:33,619 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 11:57:39,875 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/config/config.py', reloading
2023-10-17 11:57:39,875 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/config/config.py', reloading
2023-10-17 11:57:39,881 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/config/config.py', reloading
2023-10-17 11:57:40,194 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 11:57:40,610 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 11:57:40,613 - INFO - Starting the server.
2023-10-17 11:57:40,617 - WARNING -  * Debugger is active!
2023-10-17 11:57:40,617 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 12:18:57,713 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/db_test.py', reloading
2023-10-17 12:18:58,468 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 12:18:58,983 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 12:18:58,987 - INFO - Starting the server.
2023-10-17 12:18:58,992 - WARNING -  * Debugger is active!
2023-10-17 12:18:58,993 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 12:19:09,396 - INFO -  * Detected change in '/home/qwe/Pulpit/asystent/tests/db_test.py', reloading
2023-10-17 12:19:09,564 - INFO -  * Restarting with watchdog (inotify)
2023-10-17 12:19:09,905 - INFO - queries.py module loaded. Logger configuration initialized.
2023-10-17 12:19:09,907 - INFO - Starting the server.
2023-10-17 12:19:09,910 - WARNING -  * Debugger is active!
2023-10-17 12:19:09,910 - INFO -  * Debugger PIN: 685-590-235
2023-10-17 17:59:13,142 - INFO - Server stopped.
2023-10-17 17:59:13,387 - INFO - Server stopped.
